packages/curvenote-cli/src/submissions/submit.utils.ts

```diff
@@ -622,19 +624,19 @@ export async function createNewSubmission(
 
   submitLog.work = {
     id: work.id,
-    date_created: workResp?.date_created ?? work.dateCreated,
```

Comment:
The case change here, is that going to change the case of any of the fields in the SubmitLog and so potentially break build pages or other things dependent on the job payload/result blobs?

Reply:
seems like it is independent of this: https://github.com/curvenote/curvenote/blob/f0eb01b1c8a5ad11a4fbcf00e46c9dda0a0c7b64/packages/curvenote-cli/src/submissions/submit.ts#L273 and https://github.com/curvenote/curvenote/blob/f0eb01b1c8a5ad11a4fbcf00e46c9dda0a0c7b64/packages/curvenote-cli/src/submissions/submit.utils.ts#L627

Reply:
There should be no changes to the `submitLog` and therefore what is going to the `job` - the change is on the interface of the data we are _feeding into_ the log - i.e. we pull id and versions from api responses, rather from an intermediate remapped structure.